### PR TITLE
Event

### PR DESCRIPTION
**Pull request by Thommyaso**

### CODE REVIEW

File: src/lib/views/formView.js [hunk 1]
```diff
@@ -9,21 +9,43 @@ class FormView extends AbstractView {
     }
 
     init() {
+        const data = {class: 'container__alert-active'};
+        const customEvent = new CustomEvent('onValidation', {detail: data});
```
Comment (zniszcz) [inline src/lib/views/formView.js:13]: move it to click handler

File: src/lib/views/formView.js [hunk 2]
```diff
@@ -9,21 +9,43 @@ class FormView extends AbstractView {
     }
 
     init() {
+        const data = {class: 'container__alert-active'};
```
Comment (zniszcz) [inline src/lib/views/formView.js:12]: It should be only information about event, not the implementation detail

File: src/lib/views/formView.js (hunk 1 above)
Comment (zniszcz) [reply src/lib/views/formView.js:13]: You could handle only onError state

File: src/lib/views/containerView.js [hunk 3]
```diff
@@ -6,6 +6,19 @@ import TaskController from '../controllers/taskController';
 class ContainerView extends AbstractView {
     constructor(model) {
         super(model);
+
+        this.messageEl = null;
```
Comment (zniszcz) [inline src/lib/views/containerView.js:10]: It should be separated component

**Review by zniszcz — CHANGES_REQUESTED** [review]

**Comment by Thommyaso** [comment]
Closing this pull request because communication between objects in this project requires adjustment

**Pull request closed without merging**